shared/src/lib/utils.js

```diff
@@ -95,19 +97,41 @@ export async function summarizeContent({
   };
 }
 
-export async function updateSettings(handleGetData) {
+export async function fetchSettings() {
```

Comment:
Made this function more "standard" (not taking a callback, returning the object).